commands/src/main/java/org/jclouds/karaf/commands/table/internal/OSGiScriptEngine.java

```diff
@@ -0,0 +1,79 @@
+/*
```

Comment:
Do you plan to re-style the classes to align them to our coding style and change them to cleanup the code? Or do you think it makes sense to leave them unchanged in their own package?